src/static/riot/competitions/detail/submission_modal.tag

```diff
@@ -35,20 +40,28 @@
         <div class="ui grid">
             <div class="three wide column">
                 <div class="ui fluid vertical secondary menu">
-                    <div class="active submission-modal item" data-tab="{admin_: submission.admin}prediction">Prediction Logs</div>
+                    <div class="active submission-modal item" data-tab="{admin_: submission.admin}prediction">Prediction
+                        Logs
+                    </div>
                     <div class="submission-modal item" data-tab="{admin_: submission.admin}scoring">Scoring Logs</div>
                 </div>
             </div>
             <div class="thirteen wide column">
                 <div class="ui active tab" data-tab="{admin_: submission.admin}prediction">
                     <div class="ui top attached inverted pointing menu">
-                        <div class="active submission-modal item" data-tab="{admin_: submission.admin}p_stdout">stdout</div>
+                        <div class="active submission-modal item" data-tab="{admin_: submission.admin}p_stdout">stdout
```

Comment:
make these labels' indentation consistent with the ones below "Ingestion stdout"